test/unit/models/has_content_id_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class HasContentIdTest < ActiveSupport::TestCase
+  class TestObject
+    attr_accessor :content_id
+    def self.before_validation(*args)
+    end
+    def self.validates(*args)
+    end
+  end
+
+  test "it sets up generate content id" do
+    t_obj = TestObject.new
```

Comment:
At first glance I thought these variables were named with Hungarian notation. Could we use `object` or just something unabbreviated?